pay-id-api-spec/pay-id.v1.yml

```diff
@@ -11,12 +11,19 @@ paths:
         required: true
     get:
       summary: get-pay-id
+      # consumes:
+      #   - application/xrpl-mainnet+json
+      # produces:
+      #   - application/xrpl-mainnet+json
       tags: []
       responses:
         '200':
           description: OK
           content:
-            application/json:
+            application/xrpl-mainnet+json:
```

Comment:
what about devnet?

Reply:
Added.

Reply:
Maybe add `application/xrpl+json` as well (although it's shorthand for `mainnet`.

Reply:
I walked this back since we removed from spec.